app/grants/views.py

```diff
@@ -117,19 +120,23 @@ def grant_fund(request, grant_id):
     # make sure a user can only create one subscription per grant
     if request.method == 'POST':
         subscription = Subscription()
-        # subscriptionHash and ContributorSignature will be given from smartcontracts and web3
-        # subscription.subscriptionHash = request.POST.get('input_name')
-        # subscription.contributorSignature = request.POST.get('description')
-        # Address will come from web3 instance
-        # subscription.contributorAddress = request.POST.get('reference_url')
+
+        print("it fired")
+        print(request.POST)
+
+
+        subscription.subscription_hash = request.POST.get('subscription_hash')
```

Comment:
I know they shouldn't be getting to a point where they are POSTing without these parameters, but can you default to empty string and/or check for `if not var:` for these?  I'm thinking it might make the most sense for us to migrate this view to use django forms like the one introduced in `grants/forms.py` so we don't have to validate all of these entries in the view itself.

Reply:
Makes sense